gen/definitions/l2vpn_xconnect_group_p2p.yaml

```diff
@@ -19,6 +19,7 @@ attributes:
   - yang_name: neighbor/ipv4s/ipv4
     tf_name: ipv4_neighbors
     type: List
+    exclude_test: true
```

Comment:
Why did you add `exclude_test` here?

Reply:
I was checking while running the test cases with different scenario, now its removed

Reply:
Reason: Set request failed, got error: rpc error: code = Internal desc = gNMI
        commit-request: operation-failed: Invalid argument: Cross-connect Segment
        combination not supported:
        Cisco-IOS-XR-um-l2vpn-cfg:l2vpn/xconnect/groups/group[group-name =
        'P2P']/p2ps/p2p[p2p-xconnect-name =
        'XC']/neighbor/evpn/evi/segment-routing-services/service[vpn-id = '4600' and
        service-id = '600']/vpn-id